playground/dist/index.html

```diff
@@ -114,12 +130,23 @@ <h1 class="vqb__drop-zone__message">
       <button class="open-code__button" v-show="isCodeOpened" @click="hideCode">
         Hide code
       </button>
-      <pre v-show="isCodeOpened"><code>{{ code }}</code></pre>
-      <div class="warning-message" v-if="backendWarnings.length" v-for="warning in backendWarnings">
-        <i class="fas fa-exclamation-triangle"></i> {{ warning.message }}
-      </div>
-      <div class="error-message" v-if="backendErrors.length" v-for="error in backendErrors">
-        <i class="fas fa-exclamation-triangle"></i> {{ error.message }}
+      <div v-if="isCodeOpened" class="debug-panel">
+        <div class="warning-message" v-if="backendWarnings.length" v-for="warning in backendWarnings">
+          <i class="fas fa-exclamation-triangle"></i> {{ warning.message }}
+        </div>
+        <div class="error-message" v-if="backendErrors.length" v-for="error in backendErrors">
+          <i class="fas fa-exclamation-triangle"></i> {{ error.message }}
+        </div>
+        <div class="debug-panel__code">
+          <div class="debug-panel__code-mongo">
+            <div>Mongo</div>
+            <pre>{{ mongoQuery }}</pre>
+          </div>
+          <div class="debug-panel__code-vqb-pipeline">
+            <div>VQB pipeline</div>
```

Comment:
Nice!
I'd prefer:
- to avoid showing "VQB" to users: it's not the name of the project anymore and the acronym won't make any sense for them. "Pipeline" is good enough I think :)
- instead of Mongo: "Mongo query" or "translated pipeline" (if we plan in the future to swap translators)
- invert the order of panel: usually in apps like this, input in on the left and generated/transpiled/converted stuff is at right (see coffeescript, pug or other websites for examples)

Reply:
![image](https://user-images.githubusercontent.com/15191323/82037070-f276e280-96a1-11ea-9f33-52b6ac98c03d.png)